sqlserver/datadog_checks/sqlserver/activity.py

```diff
@@ -226,8 +226,11 @@ def _obfuscate_and_sanitize_row(self, row):
             if procedure_statement:
                 row['procedure_signature'] = compute_sql_signature(procedure_statement['query'])
         except Exception as e:
-            # obfuscation errors are relatively common so only log them during debugging
-            self.log.debug("Failed to obfuscate query: %s", e)
+            if self.check.log_unobfuscated_queries:
+                failed_obfuscation = row['text'] if row.get('is_proc', False) else row['statement_text']
+                self.log.warning("Failed to obfuscate query=[%s] | err=[%s]", failed_obfuscation, e)
```

Comment:
Just a nit/preference but I think that `failed_obfuscation` could be named a little better. 

```suggestion
                raw_query_text = row['text'] if row.get('is_proc', False) else row['statement_text']
                self.log.warning("Failed to obfuscate query=[%s] | err=[%s]", raw_query_text, e)
```

Reply:
Yes, this seems more clear.